nearcore/Cargo.toml

```diff
@@ -15,13 +15,16 @@ byteorder = "1.2"
 easy-ext = "0.2"
 chrono = { version = "0.4.4", features = ["serde"] }
 futures = "0.3"
+hyper = { version = "0.14", features = ["full"] }
```

Comment:
The downside is requiring compiling hyper and tokio on top of actix-web and actix and I'm not certain mixing both runtimes, even if just for tests, will always work together? I guess there isn't a better solution here if you can't use actix for the requests as you mention in the description, though.

Reply:
As I said in the other thread, I think there are definitely some friction points while trying to use both, but I don't really see what I can do about it either...